Dangerfile

```diff
@@ -1 +1,7 @@
 danger.import_dangerfile(github: 'RevenueCat/Dangerfile')
+
+# Check for submodules
+submodules = `git submodule status`.strip
+if !submodules.empty?
+  fail("This repository should not contain any submodules. Please remove them.")
```

Comment:
maybe explain why for the future devs that get this in 20 years

Reply:
Good point 👍

Reply:
Done in https://github.com/RevenueCat/purchases-ios/pull/5250/commits/b21b3ed95f89e4caf2dbb2a022ee962524c1bf96